platform/android/java/lib/src/org/godotengine/godot/Godot.java

```diff
@@ -747,6 +754,10 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle ic
 		return containerLayout;
 	}
 
+	public XRMode getXRMode() {
+		return this.xrMode;
+	}
```

Comment:
What's the use for this getter?

Reply:
I think this is called in the plugins? this is code from you that I've ported along for the past 3 years so might just be obsolete :)

Reply:
Removed it

Reply:
Turns out this was needed afterall: https://github.com/GodotVR/godot_arcore/blob/make_this_work/plugin/src/main/java/org/godotengine/plugin/arcore/ARCorePlugin.java#L20